### PR TITLE
Security Improvements & bugfixes

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: cmd/keycloakb/keycloak_bridge.go [hunk 1]
```diff
@@ -109,9 +109,13 @@ func main() {
 	var c = config(log.With(logger, "unit", "config"))
 	var (
 		// Component
-		httpAddr                = c.GetString("component-http-host-port")
 		authorizationConfigFile = c.GetString("authorization-file")
 
+		// Publishing
+		httpAddrInternal = c.GetString("http-host-port-internal")
+		httpAddrManagement = c.GetString("http-host-port-management")
+		httpAddrAccount = c.GetString("http-host-port-account")
```
Comment (fperot74) [inline cmd/keycloakb/keycloak_bridge.go]: For database, we are using prefixes (dbaudits-rw-host-port, dbaudits-ro-host-port, config-host-port)... I would see the same thing here : internal-http-host-port, management-http-host-port and account-http-host-port (this way, internal, management and account would be prefixes too for other parameters)

File: cmd/keycloakb/keycloak_bridge.go [hunk 2]
```diff
@@ -508,115 +523,141 @@ func main() {
...
-			debugSubroute.HandleFunc("/pprof/profile", http.HandlerFunc(pprof.Profile))
-			debugSubroute.HandleFunc("/pprof/symbol", http.HandlerFunc(pprof.Symbol))
-			debugSubroute.HandleFunc("/pprof/trace", http.HandlerFunc(pprof.Trace))
-		}
+		var route = mux.NewRouter()
 
-		c := cors.New(corsOptions)
-		errc <- http.ListenAndServe(httpAddr, c.Handler(route))
+		// Version.
+		route.Handle("/", http.HandlerFunc(makeVersion(ComponentName, ComponentID, Version, Environment, GitCommit)))
```
Comment (fperot74) [inline cmd/keycloakb/keycloak_bridge.go]: We don't want to have different responses for events, management and account ?

File: pkg/middleware/authentication_test.go [hunk 3]
```diff
@@ -19,14 +19,14 @@ import (
...
...NDNmNS04MDQ5LThhOTQ5MjMyZWQwYSIsInR5cCI6IkJlYXJlciIsImF6cCI6ImFkbWluLWNsaSIsImF1dGhfdGltZSI6MCwic2Vzc2lvbl9zdGF0ZSI6IjVhNTU0MDM4LWExMWQtNDM4ZC1iYmFjLTE2NzlmMWRjMmYyOSIsImFjciI6IjEiLCJyZXNvdXJjZV9hY2Nlc3MiOnsidGVzdC1yZWFsbSI6eyJyb2xlcyI6WyJ2aWV3LXJlYWxtIiwidmlldy1pZGVudGl0eS1wcm92aWRlcnMiLCJtYW5hZ2UtaWRlbnRpdHktcHJvdmlkZXJzIiwiaW1wZXJzb25hdGlvbiIsImNyZWF0ZS1jbGllbnQiLCJtYW5hZ2UtdXNlcnMiLCJxdWVyeS1yZWFsbXMiLCJ2aWV3LWF1dGhvcml6YXRpb24iLCJxdWVyeS1jbGllbnRzIiwicXVlcnktdXNlcnMiLCJtYW5hZ2UtZXZlbnRzIiwibWFuYWdlLXJlYWxtIiwidmlldy1ldmVudHMiLCJ2aWV3LXVzZXJzIiwidmlldy1jbGllbnRzIiwibWFuYWdlLWF1dGhvcml6YXRpb24iLCJtYW5hZ2UtY2xpZW50cyIsInF1ZXJ5LWdyb3VwcyJdfX0sInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZ3JvdXBzIGVtYWlsIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJncm91cHMiOlsiL3RvZV9hZG1pbmlzdHJhdG9yIl0sInByZWZlcnJlZF91c2VybmFtZSI6ImFkbWluIiwiZW1haWwiOiJ2aW5jZW50LmxvdXBAZWxjYS5jaCJ9.he9Z1NYRnqgNMzD2lSQ8yTHW8WydtxbO-wa3pzhDv-UmczT1dzoSzreuttmwrabl0CD1T0m0gZP89AI8ePdTsTOw71e7Pj8AAuqrsNs1nyhnoQlkngs2KgJZ35N3loNRs1HDyT7gTKzonJ0RkhLmKFNEK55MgNk-Nstq9LBgBR8L6jj7chcccTsYATwEfxtul-CVzvgThFYOfeWmFpPGnLdxc7o16RF_-5unqjsT3uzVvs9s7MFBM_y0yaIqjj4KkQScAi4DutPjVMVIy4WmdFn9vSGQPhUXD1DFC5CTQ3C4Po545Kq_y0IE7sRTOucBm9c1IbJhS3s0HvMVPECI0Q"
```
Comment (fperot74) [inline pkg/middleware/authentication_test.go]: Token contains a valid email... Could we create a token with a fake email address ?

Comment (harture) [reply pkg/middleware/authentication_test.go]: Bien vu

File: cmd/keycloakb/keycloak_bridge.go (hunk 1 above)
Comment (harture) [reply cmd/keycloakb/keycloak_bridge.go]: Done

**Pull request merged**